Vizzuality/heco-invest
# Upgrade the front-end application to Yarn 3 and Next.js 12

This PR upgrades the front-end application to Yarn 3 and Next.js 12.

In order to upgrade Next.js, several additional changes have been made:
- The SVG files imported directly in components are now handled by the [SVGR](https://react-svgr.com/docs/what-is-svgr/)
- Storybook now uses Webpack 5 in order to use SVGR as well
- Custom Next.js types have been written to properly type the imported SVG files

## Testing instructions

1. Run the dev server: `yarn dev`

Everything should be working fine.

2. Run Storybook: `yarn storybook:dev`

Everything should be working fine as well.

## Tracking

[LET-78](https://vizzuality.atlassian.net/browse/LET-78).
